mlflow/store/entities/paginated_collection.py

```diff
@@ -0,0 +1,4 @@
+from collections import namedtuple
+
+
+PagedReturn = namedtuple("PagedReturn", "payload page_token")
```

Comment:
Can we use `PagedList`?

https://github.com/mlflow/mlflow/blob/582377646f8d56141cfe307abb99f1b27a1962d8/mlflow/store/entities/paged_list.py#L6